bin/validate-templates.sh

```diff
@@ -109,14 +119,18 @@ function validate_templates() {
   return 0
 }
 
-if [ -n "$1" ]; then
-  [ -n "$VERBOSE" ] && echo "Running validate-templates for target cluster only"
-  validate_templates
-  # re-enable next line after helm does not throw error any more: https://github.com/helm/helm/issues/8596
-  # hf lint
-else
-  [ -n "$VERBOSE" ] && echo "Running validate-templates for all clusters"
-  for_each_cluster validate_templates
-  # re-enable next line after helm does not throw error any more: https://github.com/helm/helm/issues/8596
-  # for_each_cluster hf lint
+function main() {
+  parse_args "$@"
+  [[ $all && $label ]] && echo "Error: cannot specify --all and --label simultaneously" && exit 6
+  if [[ $all = 'y' || $label ]]; then
```

Comment:
its just `--all` right? so why check for `y`?

Reply:
`parse_args` will set `all='y'` if `--all` is passed. This will provide a "module"-like functionality in `common.sh`. But I could also set it to `'true'` or `1`, what do you prefer?

Reply:
I thought we agreed on `'true'` and `'false'` but can't find it easily in the style guide..

Reply:
In some places we use `"true"` and in other places it is `true`
So even that style is not consistent

Reply:
If we want a consistent testing approach that can handle both booleans as well as strings, and looking at your below exmple `[[ $all && $label ]]` that uses that, I would stick with the strings.

Reply:
so use 'true' and 'false'